packages/paste-website/src/components/component-status/PeerReviewStatus.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import {SuccessIcon} from '@twilio-paste/icons/esm/SuccessIcon';
+
+interface PeerReviewStatusProps {
```

Comment:
Tldr: interfaces are extensible, so in the spirit of code being sharable interfaces should be used most of the time in case they can be used for something else in the future. Interfaces can also contain a group of types.

Types are mainly used for typing a particular thing, i.e. `type StrOrNum = string|number;`.